tools/value-generation/gen.py

```diff
@@ -102,11 +139,12 @@ def processCluster(crd_value, value_map, comment_map) :
     if expectedKey not in value_map[crd_value]:
       value_map[crd_value][expectedKey] = {}
 
-  value_map[crd_value]['backup']['image'] = 'couchbase/operator-backup:1.0.0'
+  value_map[crd_value]['backup']['image'] = 'couchbase/operator-backup:1.1.0'
   value_map[crd_value]['backup']['managed'] = True
 
   value_map[crd_value]['buckets']['managed'] = True
   value_map[crd_value]['image'] = 'couchbase/server:6.6.2'
```

Comment:
Should we be setting to 7.0.2 with the scope processing as well? Probably not to align with 2.2.1.

Might want to consider 6.6.3 though?

Reply:
6.6.3 sounds legit